src/icon-manifest/supportedFolders.ts

```diff
@@ -11,7 +11,7 @@ export const extensions: IFolderCollection = {
     { icon: 'app', extensions: ['app', '.app'], format: FileFormat.svg },
     {
       icon: 'asset',
-      extensions: ['assets', '.assets'],
+      extensions: ['assets', '.assets', 'files'],
```

Comment:
I'm not sure of how `files` alias relates to `assets`. What's the rationale? I see this more of a subfolder of assets.

Reply:
On some projects I used to use files in place of assets, but I agree it can/should be a subfolder of a more generic assets folder.

Reply:
I mean, don't get me wrong, it can be whatever you want, but for that we have customization support. We only tend to implement generalized usages.

Reply:
@mgcrea if you change this, we'll include it in the next release for tomorrow.